packages/web/components/Setup/SetupDone.tsx

```diff
@@ -20,10 +21,14 @@ export const SetupDone: React.FC = () => {
       >
         {user?.player && <PlayerTile player={user.player} />}
         <MetaButton
-          onClick={() => router.push('/')}
+          onClick={() => {
+            setLoading(true);
```

Comment:
¿Have you seen a delay on this component updating state?

I'm pretty sure this update is fast enough that we don't need the `loading` check, but it shouldn't hurt anything to have this.

Reply:
Yeah. The delay is noticeable when testing in `dev` mode, but it won't be in prod.
Like you said though, it won't hurt to keep it.